File: Greedy Algorithms/Minimum cost to make array size 1 by removing larger of pairs.java
import java.util.*;

class Main{
    static int minimumCost(int[] arr){
        int size=arr.length;
        int minimumValue=arr[0];
        for(int i=1;i<size;i++){
            if(arr[i]<minimumValue)
                minimumValue=arr[i];
        }

        return (size-1)*minimumValue;
    }
    public static void main(String[] args){
        int[] arr={4,3,2};
        System.out.println(minimumCost(arr));
    }
}

File: Greedy Algorithms/Minimum cost to make array size 1 by removing larger of pairs.cpp
#include<bits/stdc++.h>
using namespace std;

int minimumCost(vector<int>& arr){
    int size=arr.size();
    int minimumValue=*min_element(arr.begin(),arr.end());
    return (size-1)*minimumValue;
}
int main(){
    vector<int> arr={4,3,2};
    cout<<minimumCost(arr);
    return 0;
}
